src/net.cpp

```diff
@@ -2345,6 +2342,11 @@ bool CConnman::Start(CScheduler& scheduler, const Options& connOptions)
     // Dump network addresses
     scheduler.scheduleEvery([this] { DumpAddresses(); }, DUMP_PEERS_INTERVAL);
 
+    // Periodically attempt to broadcast transactions that have not yet been acknowledged by the network
+    // To ensure impossibility of network-wide synchronization event, stagger retry intervals between 10-15 minutes.
+    std::chrono::seconds reattempt_broadcast_interval = std::chrono::seconds(10*60 + GetRand(5*60));
+    scheduler.scheduleEvery([this]{this->AttemptInitialBroadcast();}, std::chrono::milliseconds(reattempt_broadcast_interval));
```

Comment:
See https://github.com/ariard/bitcoin/commit/2d8ae9a838f5c6ec43db15a7bd36554c351b7442, I find a bit weird to schedule a mempool task from connection manager instead of message processing layer. It should achieve the same goal while removing one new method (or have you already considered this and turn away for a good reason) ?

Reply:
this is awesome! thanks for the patch. I was also realizing there's a simpler flow via `RelayTransaction`, but I agree that kicking off the process in `PeerLogicValidation` constructor makes more sense conceptually than `CConnman::Start`. Also I like your rename to `ReattemptInitialBroadcast`, small change but makes intentions more obvious. Thank you! 🙏🏽

Reply:
Indeed nice patch @ariard

Reply:
incorporated. thanks again!